README.md

```diff
@@ -366,6 +366,11 @@ Aras Pranckevičius' code for his series on [Daily Path Tracer experiments with
 
 ![ToyPathTracer screenshot](https://github.com/aras-p/ToyPathTracer/blob/main/Shots/screenshot.jpg?raw=true).
 
+### [Mastering Graphics Programming with Vulkan](https://github.com/PacktPublishing/Mastering-Graphics-Programming-with-Vulkan)
+Marco Castorina and Gabriel Sassone's book on developing a modern rendering engine from first principles to state-of-the-art techniques. Source code uses enkiTS.
```

Comment:
If possible, it might be better to rephrase it as:
```
Marco Castorina and Gabriel Sassone's book on developing a modern rendering engine from first principles using the Vulkan API. enkiTS is used as the task library to distribute work across cores.
```

Reply:
Done!